stable/artifactory-ha/values.yaml

```diff
@@ -1570,3 +1570,366 @@ filebeat:
     output:
       logstash:
          hosts: ["{{ .Values.filebeat.logstashUrl }}"]
+
+## Prometheus Exporter / Metrics
+##
+metrics:
+  enabled: false
+
+  service:
+    port: 24231
+    type: ClusterIP
+
+  ## Prometheus Operator ServiceMonitor configuration
+  ##
+  serviceMonitor:
+    labels: {}
+
+    ## Interval at which metrics should be scraped
+    ## ref: https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#endpoint
+    ##
+    interval: 15s
+
+    ## Timeout after which the scrape is ended
+    ## ref: https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#endpoint
+    ##
+    scrapeTimeout: 15s
+
+  fluentd:
+    image:
+      ## The Bitnami Fluentd image has the Prometheus plugin pre-installed.
+      ##
+      repository: docker.io/bitnami/fluentd
```

Comment:
Use `docker.bintray.io` please

Reply:
I tried searching for `bitnami/fluentd` and couldn't find it, only a few other bitnami images pushed by either JFrog or VMware.  I also thought this would be fine because the Filebeat container is pulling from the Elastic Docker registry.  I'm fine with making this this change but did you want me to set up this image in Bintray?

Reply:
mmm I think it should be there, right @chukka ?

Reply:
Please try to pull that image down. If it doesn't work, ignore my comment :)

Reply:
I tried with both `docker.bintray.io/bitnami/fluentd:latest` and `docker.bintray.io/bitnami/fluentd:1.11.2` and neither worked so I'm going to leave this for now.

Reply:
ok, sure. Thanks for trying